packages/create-app/templates/default-app/app-config.yaml.hbs

```diff
@@ -30,6 +30,12 @@ backend:
       password:
         $secret:
           env: POSTGRES_PASSWORD
+      ssl:
+        #ca: # if you have a CA file and want to verify it you can uncomment this section
+        #  $secret:
+        #    file: <file-path>/ca/server.crt
+        $secret:
```

Comment:
Curious. Will this throw an error if the environment variable is not provided?

I'm wondering if it might make more sense to just leave a general comment at the top of the `backend.database.connection` key which points to a docs file that describes all the available settings. That way this template never gets old assuming the docs url hasn't changed.

Reply:
It doesn’t throw an error.

Possibly but I would think that most people would be running their Postgres over ssl at least in non local envs so it would make sense for this to be made easy?

Reply:
Great, figured it was safe, but thought I would ask. Seems like the available options are listed here https://node-postgres.com/api/client and specifically for `ssl` here https://node-postgres.com/features/ssl. Although from the docs here it's not clear how you might configure this as an environment variable `POSTGRES_SSL_OPTIONS`. Maybe an example would also be useful here along with links to the documented options.